cypress-io/cypress
# chore: re-name dashboard references to Cypress Cloud

- Closes https://github.com/cypress-io/cypress/issues/24386

### User facing changelog


n/a

### Additional details


We are now referring to dashboard as Cypress Cloud. This PR updates all of the wording around the dashboard to use "Cypress Cloud" instead. I also updated some internal variable names to say "cloud" rather than "dashboard".

### Steps to test


The most important thing to test is that all Cypress Cloud actions still work with these changes. Verify the following:
- You can still log into Cypress Cloud
- The "connect project" modal flow still works
- Runs tab still links to the correct runs
- Latest runs, flaky badges, and average duration still appear on the specs list
- Other cloud integration things that I'm forgetting

Also verify that all visible copy refers to Cypress Cloud (not dashboard) and follows the [brand standards](https://docs.google.com/presentation/d/1DP2cAv0Kb9JyVD3odu4jz0vMWFzuhUEpG_AYhyBZuWk/edit#slide=id.p)

### PR Tasks


- [na] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [x] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? https://github.com/cypress-io/cypress-documentation/pull/4719
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
